packages/ag-charts-community/src/chart/galleryExamplesUSTimezone.test.ts

```diff
@@ -50,15 +50,21 @@ describe('Gallery Examples (US TZ)', () => {
     });
 
     const ctx = setupMockCanvas();
-    for (const [exampleName, example] of Object.entries(TIME_AXIS_EXAMPLES)) {
-        it(`for ${exampleName} it should create chart instance as expected`, async () => {
+    // Missing time axis examples
+    it.skip.each(Object.entries(TIME_AXIS_EXAMPLES))(
```

Comment:
why skipped?

Reply:
There weren't any examples. It was missed before in the for loop, but now jest throws an error